pil-analyzer/src/evaluator.rs

```diff
@@ -304,7 +392,9 @@ mod internal {
         Ok(match expr {
             Expression::Reference(reference) => evaluate_reference(reference, locals, symbols)?,
             Expression::PublicReference(name) => symbols.lookup_public_reference(name)?,
-            Expression::Number(n) => Value::Number(*n),
+            // TODO Default is to convert literals to integers.
```

Comment:
Will not be done in this PR.

Reply:
please open an issue so we don't forget

Reply:
https://github.com/powdr-labs/powdr/issues/947